README.md

```diff
@@ -36,21 +222,48 @@ This project is covered by the [Apache License 2.0][license-apache] also
 ## Contact
 
 The project is a collaborative effort under the umbrella of [ELIXIR Cloud &
-AAI][org-elixir-cloud]. Follow the link to get in touch with us via chat or
-email. Please mention the name of this service for any inquiry, proposal,
-question etc.
+AAI][elixir-cloud]. Follow the link to get in touch with us via chat or email.
+Please mention the name of this service for any inquiry, proposal, question
+etc. Alternatively, you can also make use of the [issue
+tracker][trs-filer-issues] to request features or report bugs.
+
+![Logo_banner][img-logo-banner]
 
-[badge-build-status]:<https://travis-ci.com/elixir-cloud-aai/trs-filer.svg?branch=dev>
-[badge-coverage]:<https://img.shields.io/coveralls/github/elixir-cloud-aai/trs-filer>
-[badge-github-tag]:<https://img.shields.io/github/v/tag/elixir-cloud-aai/trs-filer?color=C39BD3>
+[badge-build-status]:<https://travis-ci.com/elixir-cloud-aai/cloud-registry.svg?branch=dev>
+[badge-coverage]:<https://img.shields.io/coveralls/github/elixir-cloud-aai/cloud-registry>
 [badge-license]:<https://img.shields.io/badge/license-Apache%202.0-blue.svg>
-[badge-url-build-status]:<https://travis-ci.com/elixir-cloud-aai/trs-filer>
-[badge-url-coverage]:<https://coveralls.io/github/elixir-cloud-aai/trs-filer>
-[badge-url-github-tag]:<https://github.com/elixir-cloud-aai/trs-filer/releases>
+[badge-url-build-status]:<https://travis-ci.com/elixir-cloud-aai/cloud-registry>
+[badge-url-coverage]:<https://coveralls.io/github/elixir-cloud-aai/cloud-registry>
```

Comment:
@kushagra189 these links should point to `trs-filer`, but are pointing to `cloud-registry`.

Reply:
Will check and fix by today.